src/APIResponder.jl

```diff
@@ -116,6 +122,8 @@ function process(conn::APIResponder)
             respond(conn, Nullable(conn.endpoints[cmd]), :invalid_data)
         end
     end
+    close(api.sock)
+    close(api.ctx)
```

Comment:
should not close context that is not owned by itself, since caller may have passed context to constructor